config/crd/kustomization.yaml

```diff
@@ -4,16 +4,21 @@
 resources:
 - bases/opentelemetry.io_opentelemetrycollectors.yaml
 - bases/opentelemetry.io_instrumentations.yaml
+- bases/opentelemetry.io_opampbridges.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
-patchesStrategicMerge:
 # patches here are for enabling the conversion webhook for each CRD
 # +kubebuilder:scaffold:crdkustomizewebhookpatch
 
 # patches here are for enabling the CA injection for each CRD
+patchesStrategicMerge:
+# - patches/webhook_in_opampbridges.yaml
 - patches/cainjection_in_opentelemetrycollectors.yaml
+- patches/cainjection_in_opampbridges.yaml
 # +kubebuilder:scaffold:crdkustomizecainjectionpatch
 
 # the following config is for teaching kustomize how to do kustomization for CRDs.
 configurations:
 - kustomizeconfig.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
im not super familiar with what this file _should_ look like... @pavolloffay any thoughts here?